src/main/java/dmdev/vastakhov/homeworks/javacore2/lesson5/universe/Planet.java

```diff
@@ -0,0 +1,30 @@
+package dmdev.vastakhov.homeworks.javacore2.lesson5.universe;
+
+public class Planet extends MovingCelestialBody implements Flyable{
+
+    private boolean isGasPlanet;
+
+    public Planet(String name, double radius, double speed, boolean isGasPlanet) {
+        super(name, radius, speed);
+        this.isGasPlanet = isGasPlanet;
+    }
+
+    @Override
+    public String toString() {
+        return "Planet{" +
+                "isGasPlanet=" + isGasPlanet +
+                ", speed=" + speed +
+                ", name='" + name + '\'' +
+                ", radius=" + radius +
+                '}';
+    }
+
+    public String getName() {
+        return super.name;
```

Comment:
Seems that MovingCelestialBody doesn't implement getName method.
There is no need to override method if your implementation is the same as parent's